test/bun.js/serve.test.ts

```diff
@@ -5,161 +5,145 @@ import { resolve } from "path";
 
 afterEach(() => Bun.gc(true));
 
-var port = 40001;
+var port = 10000;
+var count = 200;
 
-class TestPass extends Error {
-  constructor(message) {
-    super(message);
-    this.name = "TestPass";
+async function runTest(serverOptions, test) {
+  var server;
+  while (true) {
+    try {
+      serverOptions.port = port++;
+      server = serve(serverOptions);
```

Comment:
We could use `server.reload` here and then an `afterAll` to terminate the server at the very end